### PR TITLE
ACCUMULO-2972: fixed description for instance.secret

### PR DESCRIPTION
**Pull request by milleruntime**
1.6 documentation fix for ACCUMULO-2972

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/conf/Property.java [hunk 1]
```diff
@@ -123,8 +123,9 @@
   @Sensitive
   INSTANCE_SECRET("instance.secret", "DEFAULT", PropertyType.STRING,
       "A secret unique to a given instance that all servers must know in order to communicate with one another."
-          + " Change it before initialization. To change it later use ./bin/accumulo accumulo.server.util.ChangeSecret [oldpasswd] [newpasswd], "
-          + " and then update conf/accumulo-site.xml everywhere."),
+          + "Change it before initialization. To change it later use the ChangeSecret tool and then update conf/accumulo-site.xml everywhere."
+          + "Before using the ChangeSecret tool, make sure Accumulo is not running and you are logged in as the user that controls Accumulo files in HDFS."
+          + "To use the ChangeSecret tool, run the command: ./bin/accumulo accumulo.server.util.ChangeSecret"),
```
Comment (ctubbsii) [inline core/src/main/java/org/apache/accumulo/core/conf/Property.java]: package name is incorrect here. can probably fix that

File: core/src/main/java/org/apache/accumulo/core/conf/Property.java [hunk 2]
```diff
@@ -123,8 +123,9 @@
   @Sensitive
   INSTANCE_SECRET("instance.secret", "DEFAULT", PropertyType.STRING,
       "A secret unique to a given instance that all servers must know in order to communicate with one another."
-          + " Change it before initialization. To change it later use ./bin/accumulo accumulo.server.util.ChangeSecret [oldpasswd] [newpasswd], "
-          + " and then update conf/accumulo-site.xml everywhere."),
+          + "Change it before initialization. To change it later use the ChangeSecret tool and then update conf/accumulo-site.xml everywhere."
```
Comment (ctubbsii) [inline core/src/main/java/org/apache/accumulo/core/conf/Property.java]: maybe `s/Change it before initialization/It should be changed prior to initializing Accumulo/` ?

[nits] `s/later use/later, use/`, `s/and then/and/`

File: core/src/main/java/org/apache/accumulo/core/conf/Property.java (hunk 1 above)
Comment (milleruntime) [inline core/src/main/java/org/apache/accumulo/core/conf/Property.java]: Yeah good catch.

**Comment by milleruntime** [comment]
Made English better in 0ecc47161cc5b019f856dccd78bec48c4ba70e13

**Pull request merged**